CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+# Contributing to CONTRIBUTING.md
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+
+## Table of Contents
+
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting Enhancements](#suggesting-enhancements)
+<!-- TODO
+- [Your First Code Contribution](#your-first-code-contribution)
+- [Improving The Documentation](#improving-the-documentation)
+-->
+- [Styleguides](#styleguides)
+- [Commit Messages](#commit-messages)
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation](docs/).
+
+Before you ask a question, it is best to search for existing [Issues](/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
```

Comment:
The issues link may be broken

Reply:
maybe is better to link `https://github.com/lambdaclass/cairo-rs/issues` ?

Reply:
It's relative to the repository, so it should work. I can test it to be sure.

Reply:
Huh, it was indeed broken, nice catch!

Reply:
Now all links work.